common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -257,7 +257,18 @@ def student_view(self, context):
                 banner_text, special_html = special_html_view
                 if special_html and not masquerading_as_specific_student:
                     return Fragment(special_html)
-        return self._student_view(context, prereq_met, prereq_meta_info, banner_text)
+        return self._student_or_public_view(context, prereq_met, prereq_meta_info, banner_text)
+
+    def public_view(self, context):
+        """
+        Renders the preview view of the block in the LMS.
+        """
+        prereq_met = True
+        prereq_meta_info = {}
+
+        if self._required_prereq():
+            prereq_met, prereq_meta_info = self._compute_is_prereq_met(True)
```

Comment:
@ormsbee I had to put this in during the rebase to match recent changes to the `student_view`: https://github.com/edx/edx-platform/commit/6fbd86108ce07484ce95dcc0df5ee836c40f8ccd#diff-14daf51ec8decc308953f7a4e7e0a03f